### ISSUE TITLE
Duplicate cars

### ISSUE THREAD

**Issue by KilluaZoldyck0099** [TanguyOrtegat/esx_jb_eden_garage#2]
Has anyone fixed duplication of cars bug?

**Comment by TanguyOrtegat** [comment]
yet i do only update garage 2 but feel free to implement same logic as garage 2 :)

**Comment by KilluaZoldyck0099** [comment]
Yes but, Garage 2 is not in English.

**Comment by TanguyOrtegat** [comment]
yes so ? you cannot translate it or is ti too much work as developper ?

**Comment by KilluaZoldyck0099** [comment]
I’v alread Done but I still can put out two cars

**Comment by Abrahamarmas** [comment]
can anyone of you guys would be a good person and gave me the esx_garage2 so i can test it out </3
im new at programing and need some help

**Comment by nickglas** [comment]
i also can duplicate cars. needs to be fixed but idk how.

**Comment by TanguyOrtegat** [comment]
take garage 2 this garage needs some update but have no time to fix it
